package/utils/px5g-wolfssl/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2020 Paul Spooren <mail@aparcar.org>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=px5g-wolfssl
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-2.0-or-later
+
+PKG_USE_MIPS16:=0
+
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/px5g-wolfssl
+  SECTION:=utils
+  CATEGORY:=Utilities
+  SUBMENU:=Encryption
+  TITLE:=X.509 certificate generator (using wolfssl)
+  DEPENDS:=+libwolfssl +@CONFIG_WOLFSSL_HAS_CERTGEN
```

Comment:
This will not work. `+` is not supported for configuration symbols and you have to drop the `CONFIG_` prefix when using the `@` syntax. Make it `@WOLFSSL_HAS_CERTGEN` instead and select that symbol by default in the wolfssl makefile.

Reply:
Done